test/e2e/run_test.go

```diff
@@ -1615,9 +1615,12 @@ VOLUME %s`, ALPINE, volPath, volPath)
 		// Run and replace 5 times in a row the "same" container.
 		ctrName := "testCtr"
 		for i := 0; i < 5; i++ {
-			session := podmanTest.Podman([]string{"run", "--detach", "--replace", "--name", ctrName, ALPINE, "/bin/sh"})
+			session := podmanTest.Podman([]string{"run", "--detach", "--replace", "--name", ctrName, ALPINE, "top"})
 			session.WaitWithDefaultTimeout()
 			Expect(session).Should(ExitCleanly())
+
+			// make sure Podman prints only one ID
+			Expect(session.OutputToStringArray()).To(HaveLen(1))
```

Comment:
Nit: In case you have to repush, I'd like to check that it's really the CID.

Reply:
I'll second this. Very easy fix:
```diff
diff --git a/test/e2e/run_test.go b/test/e2e/run_test.go
index 036cd4569..423c2e44c 100644
--- a/test/e2e/run_test.go
+++ b/test/e2e/run_test.go
@@ -1622,3 +1622,3 @@ VOLUME %s`, ALPINE, volPath, volPath)
 			// make sure Podman prints only one ID
-			Expect(session.OutputToStringArray()).To(HaveLen(1))
+			Expect(session.OutputToString()).To(HaveLen(64))
 		}
```